DogTales/src/components/physics.hpp

```diff
@@ -4,25 +4,31 @@
 
 namespace component {
 
-static constexpr glm::vec2 max_velocity{1000.f, 1000.f};
-static constexpr float tick_limit{0.03f}; //this should possibly go somewhere else and be globally accessible
+constexpr glm::vec2 default_friction{0.999f};
+constexpr float default_gravity{-1000.f};
+constexpr float default_mass{1.f};
+
+constexpr glm::vec2 max_velocity_v{1000.f};
+constexpr bave::Seconds tick_limit_v{100ms};
```

Comment:
I'd move these constants into `class Physics` (as `static constexpr`) class since they are specific to it.

Reply:
fixed